web/locales/fi.json

```diff
@@ -0,0 +1,1786 @@
+    "electricityStoredUsing": "<b>%1$s %%</b> <img id=\"country-flag\"></img> <b>%2$s</b> saatavilla olevasta sähköstä varastoidaan %3$s",
+    "emissionsStoredUsing": "<b>%1$s %%</b> <img id=\"country-flag\"></img> <b>%2$s</b> saatavilla olevista sähkön päästöistä varastoidaan %3$s",
+
+    "electricityExportedTo": "<b>%1$s %%</b> <img id=\"country-flag\"></img> <b>%2$s</b> saatavilla olevasta sähköstä viedään %3$s",
+    "emissionsExportedTo": "<b>%1$s %%</b> <img id=\"country-flag\"></img> <b>%2$s</b> saatavilla olevista sähkön päästöistä viedään %3$s",
+    "electricityImportedFrom": "<b>%1$s %%</b> <img id=\"country-flag\"></img> <b>%2$s</b> saatavilla olevasta sähköstä tuodaan %3$s",
+    "emissionsImportedFrom": "<b>%1$s %%</b> <img id=\"country-flag\"></img> <b>%2$s</b> saatavilla olevista sähkön päästöistä tuodaan %3$s",
+
+    "ofCO2eqPerMinute": "hiilidioksiidia minuutissa",
+    "theMap": {
+        "groupName": "Kartta",
+        "mapColors-question": "Mitä kartan värit tarkoittavat?",
+        "mapColors-answer": "Väritämme alueet kartalla kasvihuonekaasumäärien perusteella, jotka päästetään jokaisesta siellä kulutetusta sähköyksiköstä (sen <a href=\"#carbonIntensity\" class=\"entry-link\">hiili-intensiteetti</a>). Mitä vihreämpi väri, sitä ilmastoystävällisempi sähkönkulutus on.",
+        "mapArrows-question": "Mitä kartan alueiden väliset nuolet osoittavat?",
+        "mapArrows-answer": "Alueiden väliset nuolet osoittavat sähkön fyysisen virtauksen (tuonnit ja viennit) alueiden välillä: mitä nopeammin ne vilkkuvat, sitä nopeampi virtaus on. Sähkövirta on tärkeä, koska kun tuot osaa sähköstä naapurialueilta, tuot myös sen tuotantoon liittyviä hiilidioksidipäästöjä.",
+        "mapSolarWindButtons-question": "Mitä kartalla olevat aurinko- ja tuulipainikkeet tekevät?",
+        "mapSolarWindButtons-answer": "Nämä painikkeet näyttävät reaaliaikaisen tuulen nopeuden ja auringonpaisteen voimakkuuden, mikä osoittaa mahdollisuuden siirtyä aurinko- ja tuulipohjaisiin energiavaihtoehtoihin eri alueilla. Huomaa, että tämä on vain pintakäsittely, koska tuuliturbiinien ja aurinkokennojen asentamisen todellinen potentiaali riippuu myös monista muista tekijöistä.",
+        "mapNuclearColors-question": "Miksi ydinkäyttöiset maat näkyvät kartalla vihreinä?",
+        "mapNuclearColors-answer": "Ydinenergian tuotannossa on erittäin alhainen hiili-intensiteetti (<a href=\"https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources\" target=\"_blank\" >lähde</a>), ja ydinvoimaloiden maat esitetään siksi kartalla vihreinä. Tämä ei tarkoita, että ydinenergiaan ei liity muita ympäristöhaasteita (kuten myös muissa energiantuotantomuodoissa), mutta nämä haasteet eivät liity kasvihuonekaasujen päästöihin, joten ne eivät kuulu tämän kartan laajuteen.",
+        "mapColorBlind-question": "Olen värisokea. Kuinka voin lukea karttaasi?",
```

Comment:
`karttaasi` is like directly addressing someone, e.g. Asking someone "Can I loan your map"? `karttaanne` would be more general asking like a wider audience.